success-criteria.md

```diff
@@ -485,7 +457,7 @@ stakeholder group.
 
 #### Interests of Publishers
 
--   Ad-funded business model to provide free access to all:
+-   The capacity to run an ad-funded business that returns a profit significant enough to provide free access at scale and continue operation:
```

Comment:
Too general. How about this replacement.

Reply:
This would benefit from a clearer definition of publishers, too. The definition that adtech has usually used for this has been "anything with inventory". I can write fart apps and torchlight apps that'll turn a profit at any scale. That's not the same as what people usually think of as publishers.

Reply:
"Publishers" is defined using the W3C terminology listed [here](https://www.w3.org/1999/05/WCA-terms/). This ensures the term "Publisher" will be familar to all W3C members across multiple documents.

I think your point is to distinguish between all publishers from quality publishers, which is the function of the healthy market operations, whereas the prupose of this document is to ensure an open market exists, and the impact on this market. This document can help authors evaluate their proposals even at the conceptual stage to ensure they're improving the web.

Reply:
Where did you find that piece of history? A 1999 working draft is probably not the most useful reference for an evolving ecosystem. It's fine to adopt new more specific definitions for a new document.

Reply:
I've seen it used as a normative source it other relatively recent W3C documents so used it to ensure terms would be familiar to W3C participants rather than redefine them. I don't recall exactly which document it was in.